docs/server/source/data-models/block-model.rst

```diff
@@ -25,66 +19,10 @@ The JSON Keys in a Block
 
 **id**
 
-The transaction ID and also the SHA3-256 hash
-of the inner ``block`` object, loosely speaking.
-It's a string.
-To compute it, 1) construct an :term:`associative array` ``d`` containing
-``block.timestamp``, ``block.transactions``, ``block.node_pubkey``,
-``block.voters``, and their values. 2) compute ``id = hash_of_aa(d)``.
-There's pseudocode for the ``hash_of_aa()`` function
-in the `IPDB Transaction Spec page about cryptographic hashes
-<https://the-ipdb-transaction-spec.readthedocs.io/en/latest/common-operations/crypto-hashes.html#computing-the-hash-of-an-associative-array>`_.
-The result (``id``) is a string: the block ID.
-An example is ``"b60adf655932bf47ef58c0bfb2dd276d4795b94346b36cbb477e10d7eb02cea8"``
+The block ID denotes the height of the blockchain when the given block was committed.
```

Comment:
Please add that `id` is an integer (or is it a string?) and that it starts at 0 (or does it start at 1?).